### PR TITLE
docs: update crd types

### PR DESCRIPTION
**Pull request by shinytang6**
Signed-off-by: shinytang6 <1074461480@qq.com>

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by cwen0** [comment]
@shinytang6 Thanks for your contribution!

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Chaos Operator injects chaos into the applications and Kubernetes infrastructure
 ![Chaos Operator](./static/chaos-mesh.svg)
 
 Chaos Operator uses [Custom Resource Definition (CRD)](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) to define chaos objects.
-The current implementation supports six types of CRD objects for fault injection, namely PodChaos, NetworkChaos, IOChaos, TimeChaos, StressChaos, and KernelChaos, which correspond to the following major actions (experiments):
+The current implementation supports a few types of CRD objects for fault injection, namely DNSChaos, HTTPChaos, PodChaos, PodIoChaos, PodNetworkChaos, NetworkChaos, IoChaos, TimeChaos, StressChaos, and KernelChaos, which correspond to the following major actions (experiments):
```
Comment (WangXiangUSTC) [inline README.md:35]: Could you please add the description of these new CRD types below?

Comment (shinytang6) [reply README.md:35]: Added some descriptions, but l'm not familiar with the specified action of each chaos, pls correct me if necessary:)

Comment (WangXiangUSTC) [reply README.md:35]: thanks, we can delete the HTTPChaos and the description about it, because it can not work now. rest LGTM

Comment (shinytang6) [reply README.md]: OK, done.

**Review by WangXiangUSTC — APPROVED** [review]
LGTM

**Review by Gallardot — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ Chaos Operator injects chaos into the applications and Kubernetes infrastructure
 ![Chaos Operator](./static/chaos-mesh.svg)
 
 Chaos Operator uses [Custom Resource Definition (CRD)](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) to define chaos objects.
-The current implementation supports six types of CRD objects for fault injection, namely PodChaos, NetworkChaos, IOChaos, TimeChaos, StressChaos, and KernelChaos, which correspond to the following major actions (experiments):
+The current implementation supports a few types of CRD objects for fault injection, namely DNSChaos, PodChaos, PodIoChaos, PodNetworkChaos, NetworkChaos, IoChaos, TimeChaos, StressChaos, and KernelChaos, which correspond to the following major actions (experiments):
```
Comment (mehmetcn) [inline README.md:35]: 100

**Pull request merged**